src/app/modules/consultations/consultation-profile/consultation-response-text/consultation-response-text.component.ts

```diff
@@ -51,42 +62,41 @@ export class ConsultationResponseTextComponent implements OnInit, AfterViewCheck
     this.getCurrentUser();
     this.subscribeUseTheResponseText();
     this.getResponseText();
-    this.scrollToCreateResponse();
```

Comment:
@Nishanthh Why is this removed?

Reply:
@avikt if we dont have a response text, triggers scroll to error and reach responsecreate container otherwise it submits the responsetext input in the container. so scrolling to response container is not needed